app/src/main/java/com/example/concordia_campus_guide/Fragments/InfoCardFragment/InfoCardFragment.java

```diff
@@ -167,8 +169,8 @@ private void setBuildingImage(String buildingCode){
                 buildingImage.setImageDrawable(d);
             }
             catch(Exception e2){
-                System.out.println(e.getMessage());
-                System.out.println(e2.getMessage());
+                Log.e("MyLog", "jpg image cannot be shown in the info card: " + e.getMessage());
```

Comment:
Instead of MyLog, I would suggest to put a more descriptive string: perhaps a string that describes the purpose of the function or the class name.